controllers/argoutil/resource.go

```diff
@@ -52,17 +52,23 @@ func CombineImageTag(img string, tag string) string {
 }
 
 // CreateEvent will create a new Kubernetes Event with the given action, message, reason and involved uid.
-func CreateEvent(client client.Client, eventType, action, message, reason string, meta metav1.ObjectMeta) error {
-	event := newEvent(meta)
+func CreateEvent(client client.Client, eventType, action, message, reason string, ObjectMeta metav1.ObjectMeta, typeMeta metav1.TypeMeta) error {
```

Comment:
I'm curious why the `ObjectMeta` parameter name starts with an uppercase letter?

Reply:
no particular reason
changed it, thanks